lib/mysql.js

```diff
@@ -448,7 +448,8 @@ MySQL.prototype.fromColumnValue = function(prop, val) {
       case 'Array':
       case 'Object':
       case 'JSON':
-        if (typeof val === 'string') {
+        // Added length check to prevent parsing errors of an empty string
+        if (typeof val === 'string' && val.length) {
```

Comment:
I feel we should try catch the error if it's an empty string. In the above case, "empty string" is nothing special than an invalid `string` argument for function `JSON.parse()`, if parsing error blows up the app, user should know the input is wrong.